test/config-luatest/credentials_applier_test.lua

```diff
@@ -1237,21 +1228,20 @@ g.test_postpone_grants_till_rename = function(g)
     helpers.success_case(g, {
         options = iconfig,
         verify = function(iconfig)
-            local internal =
-                    require('internal.config.applier.credentials')._internal
+            local internal = require('access_control')._internal
 
             local function check_sync()
 
                 local perm = box.schema.user.info('myuser')
                 perm = internal.privileges_from_box(perm)
 
                 local exp_perm = iconfig.credentials.users.myuser
-                exp_perm = internal.privileges_from_config(exp_perm)
+                exp_perm = internal.privileges_from_data(exp_perm)
```

Comment:
I have the same point as with `config_` -> `source_` replacement. It feels more natural and consistent than removing `config` in some places and replacing it with `data` in others.